src/main/java/edu/kis/vh/nursery/factory/RyhmerFactoryImpl.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery.factory;
+
+import edu.kis.vh.nursery.DefaultCountingOutRyhmer;
+import edu.kis.vh.nursery.FIFORyhmer;
+import edu.kis.vh.nursery.HanoiRyhmer;
+import edu.kis.vh.nursery.collections.intType.IntArrayStack;
+import edu.kis.vh.nursery.collections.intType.IntLinkedList;
+
+public class RyhmerFactoryImpl implements Ryhmersfactory {
```

Comment:
3.1.16 was about adding new factory classes, not new methods to the factory interface.